docs/terraformoptions.adoc

```diff
@@ -407,6 +407,16 @@ EOT
 | RUNNING/STOPPED
 | RUNNING
 
+|enable_operator_pv_encryption_in_transit
```

Comment:
Please enclose the name of a variable between backticks so it renders like `my_variable`

Reply:
Please group variables together, within each group sort them in alphabetical order

Reply:
done